src/Pages/Apartment/components/Map/MapContainer.js

```diff
@@ -0,0 +1,493 @@
+//           beforeLevel6PositionsMarkers.push(marker);
+//         }
+//         map.relayout();
+//       }
+
+//       console.log(level);
+//     });
+
+//     let beforeLevel6Positions = [
+//       {
+//         title: '건물1',
+//         minimum_price: '8.7억',
+//         maximum_price: '18.7억',
+//         latlng: new kakao.maps.LatLng(31.5063702427021, 127.373641718788),
+//       },
+//       {
+//         title: '규석이네 집',
+//         minimum_price: '3.7억',
+//         maximum_price: '12.7억',
+//         latlng: new kakao.maps.LatLng(37.5063702427023, 127.053651718758),
+//       },
+//       {
+//         title: '건물3',
+//         minimum_price: '10.7억',
+//         maximum_price: '12.7억',
+//         latlng: new kakao.maps.LatLng(37.5113712427025, 127.051653728778),
+//       },
+//       {
+//         title: '건물4',
+//         minimum_price: '3.7억',
+//         maximum_price: '12.7억',
+//         latlng: new kakao.maps.LatLng(37.5123712427025, 127.053753728778),
+//       },
+//       {
+//         title: '건물5',
+//         minimum_price: '1.7억',
+//         maximum_price: '8.7억',
+//         latlng: new kakao.maps.LatLng(37.5014602427023, 127.051641718758),
+//       },
+//       {
+//         title: '건물1',
+//         minimum_price: '9.7억',
+//         maximum_price: '50.7억',
+//         latlng: new kakao.maps.LatLng(37.5024602427023, 127.061641718758),
+//       },
+//       {
+//         title: '규석이네 별장',
+//         minimum_price: '0.2억',
+//         maximum_price: '80.7억',
+//         latlng: new kakao.maps.LatLng(37.5124602427023, 127.062641718758),
+//       },
+//       {
+//         title: '건물3',
+//         minimum_price: '8.7억',
+//         maximum_price: '18.7억',
+//         latlng: new kakao.maps.LatLng(37.5114602427023, 127.062741718758),
+//       },
+//       {
+//         title: '건물4',
+//         minimum_price: '8.7억',
+//         maximum_price: '18.7억',
+//         latlng: new kakao.maps.LatLng(37.5124602427023, 127.081741718758),
+//       },
+//       {
+//         title: '태현이 작업실',
+//         minimum_price: '8.7억',
+//         maximum_price: '18.7억',
+//         latlng: new kakao.maps.LatLng(37.5134602427023, 127.087741718758),
+//       },
+//     ];
+
+//     let afterLevel6Positions = [
+//       {
+//         title: '삼성동',
+//         latlng: new kakao.maps.LatLng(31.5063702427021, 127.373641718788),
+//         average_price: '15 억',
+//       },
+//       {
+//         title: '압구정동',
+//         latlng: new kakao.maps.LatLng(37.5063702427023, 127.053651718758),
+//         average_price: '13 억',
+//       },
+//       {
+//         title: '대치동',
+//         latlng: new kakao.maps.LatLng(37.5263702427022, 127.051641918748),
+//         average_price: '20.6 억',
+//       },
+//     ];
+//   }, [level]);
+
+//   return <MapElement id='map'></MapElement>;
+// };
+
+class MapContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      level: 5,
+    };
+  }
```

Comment:
```suggestion
state = {
      level: 5,
};
```
- props에 접근하는 것이 아니라면 constructor 함수 밖에서 state 초기값을 정의할 수 있습니다.
- 무슨 level을 의미하는건가요? key 이름 수정해주세요.

Reply:
- 오! 몰랐습니다. 메모하겠습니다!!

- 수정하겠습니다!!